build.sh

```diff
@@ -198,7 +198,7 @@ patch_osbuild() {
     # To make it easier to apply patches we'll move around the osbuild
     # code on the system first:
     rmdir /usr/lib/osbuild/osbuild
-    mv /usr/lib/python3.13/site-packages/osbuild /usr/lib/osbuild/
+    mv /usr/lib/python3.14/site-packages/osbuild /usr/lib/osbuild/
```

Comment:
we might be able to not have this problem in the future if we do some dynamic detection here:

```
python_lib_dir=$(ls -d /usr/lib/python*)
```

and then use that var in the below code.

Reply:
note.. if we merged this change first then the prow jobs mentioned in https://github.com/coreos/coreos-assembler/pull/4348#issuecomment-3504323581 would probably succeed.

However it was good that they failed so we'd know to go update them. Would you mind adding a step to that to the rebase checklist?